pkg/store/schema/00001_create_users_table.sql

```diff
@@ -1,12 +1,5 @@
--- +goose Up
--- +goose StatementBegin
+USE spiredemo;
```

Comment:
do we need to separate statements into multiple `+goose StatementBegin` sections?

Reply:
nvm, we're moving away both the scripts from goose

Reply:
I wasn't using goose anymore since DB teardown is a hard takedown of the MySQL cluster